CMakeLists.txt

```diff
@@ -364,6 +366,17 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "BSD")
 	if(${CMAKE_SYSTEM_NAME} MATCHES "FreeBSD")
 		set(FREEBSD TRUE)
 	endif()
+	if(${CMAKE_SYSTEM_NAME} MATCHES "OpenBSD")
```

Comment:
I would have moved this detection up before setting the CMAKE_C_FLAGS then it would already be possible to do a if (NOT OPENBSD) instead of if(NOT ${CMAKE_SYSTEM_NAME} MATCHES "OpenBSD"). But that's just personal flavor :)